pyuoi/linear_model/base.py

```diff
@@ -371,10 +408,10 @@ def fit(self, X, y, stratify=None, verbose=False):
                 scores[ii] = self._score_predictions(
                     metric=self.estimation_score,
                     fitter=fitter,
-                    X=np.zeros_like(X_test), y=y_test,
+                    X=np.zeros(X_test.shape, dtype=X_test.dtype), y=y_test,
```

Comment:
Can we check to see if X_test is sparse here and create a sparse matrix if so? I think creating empty sparse matrices corresponds to all zeros.

Reply:
Might be better if `X_test` is very large and sparse.